viewer/src/migration/types.ts

```diff
@@ -45,3 +45,12 @@ export interface AddModelOptions {
   onProgress?: (progress: OnProgressData) => void;
   onComplete?: () => void;
 }
+
```

Comment:
Sorry, not sure I was clear :) Since there is already a file with a type that covers this. I would want to avoid creating new. I would suggest renaming the File3DFormat to SupportedModelTypes or something like that :)